### ISSUE TITLE
One-line install of tldr, without any node server

### ISSUE THREAD

**Issue by josephernest** [tldr-pages/tldr#1588]
`tldr` looks very nice, especially

    $ tldr tar

which is a good example.

Question: is there a one-line install solution like

    apt-get install tldr-pages

i.e. for people who don't have (or don't want to install) a node server.

I imagine it's possible: after all docs can be just plain text files, and 

    $ tldr tar

could just display one of these files. 

Is there a version without the need of a node server?

**Comment by agnivade** [comment]
There are a whole plethora of clients other than node. https://github.com/tldr-pages/tldr#clients. So, yes you can have tldr without installing node.

Now, regarding a debian package, note that a debian package will only work for debian operating systems. But tldr aims to be a platform independent tool. Therefore, we would ideally want a solution which works on all platforms. Please check the discussion here - https://github.com/tldr-pages/tldr/issues/1056#issuecomment-246575809. Feel free to add your comments

To answer your question - no we do not have a debian package for tldr. But you are most welcome to create one.

**Comment by josephernest** [comment]
Thanks for your answer.

> There are a whole plethora of clients other than node

The term "client" led to a misunderstanding for me: I thought that even if I use another *client* (such as one in your list), there would still be the need of a  *server* running. (client/server). Is this true or false? 

But anyway, why is a server needed? I don't know anything about how `tldr` works internally, but isn't it possible to have just a collection of hundreds or thousands of .txt files? And then:

    tldr tar

would be just a shortcut for:

    less /usr/doc/tldr/tar.txt

Is there a good reason for not dealing with plain text files?

Again, from a Layman approach, all I expect is [something like this](https://github.com/tldr-pages/tldr/raw/master/screenshot.png), and this could be totally done with a collection of txt files.

(Or maybe there's a more complex system that I don't understand?)

**Comment by agnivade** [comment]
> The term "client" led to a misunderstanding for me: I thought that even if I use another client (such as one in your list), there would still be the need of a server running. (client/server). Is this true or false?

False. There is no server running anywhere. The "client" is just code written to display the markdown page.

>

**Comment by sbrl** [comment]
The "server" you speak of is actually the GitHub repository - The clients just pull down content from there.

**Comment by josephernest** [comment]
> The "server" you speak of is actually the GitHub repository - The clients just pull down content from there.

I don't think so: according to @agnivade said [here](https://github.com/tldr-pages/tldr/issues/1592#issuecomment-340039060), you only pull data from the repo *once*. Then everything happens on the local machine and there is no client/server request anymore each time you do `tldr <command>`. So I think it's better to say "there's no server involved" (after initial install or except updating once in a while).

**Comment by sbrl** [comment]
Actually, that's not true. Yeah, most clients cache it locally, but most periodically update (you can do it manually via `--update`). Who knows - some clients might actually make a request for every page you ask for. There no set way of doing it, and so everyone does it differently. There's another issue about that (I forget which one) and talk of an 'official' reference client.

**Comment by agnivade** [comment]
Yes, I just tried to give a overall idea without going into details. @sbrl is right.
